server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -112,7 +112,15 @@ export default class ActivitiesMigrationController {
         })
         return null
       }),
-    ]).then(([estimatedCount, dlqCount, incentiveLevels, nomisFeatureSwitchOn, dpsPrisonRollout]) => {
+
+      this.activitiesService.checkPrisonPayBandsExist(prisonId, context(res)).catch(error => {
+        errors.push({
+          text: `Failed to check if prison ${prisonId} has pay bands in DPS due to error: ${error.message}`,
+          href: '',
+        })
+        return null
+      }),
```

Comment:
The main change is to check if there are any pay bands set up for the prison in DPS.